src/httprpc.cpp

```diff
@@ -226,6 +226,11 @@ static bool InitRPCAuthentication()
     return true;
 }
 
+void RegisterJSONEndpoint(const std::string& endpoint, bool exactMatch)
```

Comment:
This seems superfluous. Why not just either auto-register based on the endpoint in the commands table or just register everything explicitly in httprpc.cpp (its only 4 things).